public/app/features/explore/state/query.ts

```diff
@@ -1328,25 +1325,12 @@ const processQueryResponse = (state: ExploreItemState, action: PayloadAction<Que
     if (error.type === DataQueryErrorType.Timeout || error.type === DataQueryErrorType.Cancelled) {
       return { ...state };
     }
-
-    // Send error to Angular editors
-    // When angularSupportEnabled is removed we can remove this code and all references to eventBridge
```

Comment:
Could you also remove all references to `eventBridge` in Explore code? It shouldn't be used anymore

Reply:
Without removal is there anything here that is breaking explore @ifrost ? I'm not very keen on making this PR any larger than it needs to be. Once this PR is merged there will be further clean up needed throughout the codebase to completely remove traces of Angular.